src/components/Text.tsx

```diff
@@ -155,11 +155,16 @@ export function BodyText({
   )
 }
 
-const headerText = (accent = false, center?: boolean, extraLeading = false) =>
+const headerText = (
+  accent = false,
```

Comment:
I think we should have an object args here, too many of them to remember the order